api/v1alpha1/mongodbcluster_types.go

```diff
@@ -51,6 +51,33 @@ type MongoDBClusterSpec struct {
 
 	// If this is set, along with useAtlasApi, all the kubernetes nodes on the cluster will be added to the Atlas firewall. The only available value right now is "rancher-annotation", which uses the rke.cattle.io/external-ip annotation.
 	AtlasNodeIPAccessStrategy string `json:"atlasNodeIpAccessStrategy,omitempty"`
+
+	AtlasScheduledAutoscaling *AtlasScheduledAutoscaling `json:"atlasScheduledAutoscaling,omitempty"`
+}
+
+type AtlasScheduledAutoscaling struct {
+	// If this is set, the cluster will be enabled for scheduled autoscaling.
+	// The way it works is that the cluster will be scaled up to the high tier at the specified time, and scaled down to the lowTier at the specified time.
+	// +kubebuilder:default=false
+	Enabled bool `json:"enabled,omitempty"`
+
+	// The minimum tier the cluster can scale down to.
+	// +kubebuilder:validation:Enum=M0;M2;M5;M10;M20;M30;M40;M50;M60;M80;M140;M200;M300;M400;M500;M700;M900;M1000
+	// +kubebuilder:default="M20"
+	LowTier string `json:"lowTier,omitempty"`
+
+	// The maximum tier the cluster can scale up to.
+	// +kubebuilder:validation:Enum=M0;M2;M5;M10;M20;M30;M40;M50;M60;M80;M140;M200;M300;M400;M500;M700;M900;M1000
```

Comment:
should shared tiers be listed as valid scale downs or scale ups? likely we don't want to scale down so much or up, we could avoid mistakes like typos by not having them in validation.